jenkins-integration/beaker/install/shared/45_install_hiera_config.rb

```diff
@@ -18,13 +20,13 @@ def get_source_hiera_config(host, hiera_config)
   File.join(local_hiera_dir, File.basename(source_hiera_config_path))
 end
 
-def update_hiera_datadir_in_local_config(host, hiera_config, local_hiera_config_path)
+def update_hiera_datadir_in_local_config(_host, hiera_config, local_hiera_config_path)
```

Comment:
:man_shrugging: This method looks dodgy to me. I presume that it _should_ take a host (since it is called with master below) and that the commands in this method should happen there.